pyOCD/pyDAPAccess/__init__.py

```diff
@@ -14,8 +14,7 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
+from .dap_access_api import DAPAccessIntf
+from .dap_access_usb import DapAccessUSB
 
-from cmsis_dap import CMSIS_DAP
-
-TRANSPORT = {'cmsis_dap': CMSIS_DAP
-            }
+DAPAccess = DapAccessUSB
```

Comment:
DAPAccess is a alias to DAPAccessUSB.  That way modules using it only need to call "from pyDAPAccess import DAPAccess".